packages/harmony/.storybook/manager-head.html

```diff
@@ -25,16 +16,20 @@
     color: var(--harmony-docs-bar-text-color) !important;
   }
   
-  .sidebar-item:hover {
-    background: var(--harmony-docs-violet-v200) !important;
+  .sidebar-item:not([data-selected="true"]):hover {
+    background: var(--harmony-docs-bar-hover-color);
   }
 
   .sidebar-item svg, .search-result-item svg {
-    color: var(--harmony-docs-violet-v200) !important;
+    color: var(--harmony-docs-icon-color) !important;
   }
 
-  .sidebar-item[data-selected="true"] {
-    background: var(--harmony-docs-violet-v400) !important;
+  /* .sidebar-item[data-selected="true"] {
+    background: var(--harmony-docs-bar-selected-color) !important;
+  } */
```

Comment:
should this be removed to keeping here for future reference?

Reply:
yes good call, removing